core/internal/gethwrappers/utils.go

```diff
@@ -0,0 +1,47 @@
+package gethwrappers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+func Exit(msg string, err error) {
+	if err != nil {
+		fmt.Println(msg+":", err)
+	} else {
+		fmt.Println(msg)
+	}
+	os.Exit(1)
+}
+
+// GetProjectRoot returns the root of the chainlink project
+func GetProjectRoot() (rootPath string) {
+	root, err := os.Getwd()
+	if err != nil {
+		Exit("could not get current working directory while seeking project root",
+			err)
+	}
+	for root != "/" { // Walk up path to find dir containing go.mod
+		if _, err := os.Stat(filepath.Join(root, "go.mod")); os.IsNotExist(err) {
+			root = filepath.Dir(root)
+		} else {
+			return root
+		}
+	}
+	Exit("could not find project root", nil)
+	panic("can't get here")
```

Comment:
What is this for?

Reply:
It satisfies staticcheck, which complains that there's no return at the end of the function.